src/Language/PureScript/Docs/Tags.hs

```diff
@@ -0,0 +1,46 @@
+module Language.PureScript.Docs.Tags
+  ( dumpCtags
+  , dumpEtags
+  ) where
+
+import Prelude
+
+import           Control.Arrow (first)
+import           Data.List (sort)
+import           Data.Maybe (mapMaybe)
+import qualified Data.Text as T
+import Language.PureScript.AST (SourceSpan, sourcePosLine, spanStart)
+import Language.PureScript.Docs.Types
+
+tags :: Module -> [(String, Int)]
+tags = map (first T.unpack) . concatMap dtags . modDeclarations
```

Comment:
this looks like a nice and pure function, what do you think about adding a few tests to make sure tag generation continues to work as we change the compiler?

Reply:
I think it's a great idea :). Actually, it's the main reason I proposed moving this code in the first place.